### DISCUSSION TITLE
Yet Another Broken Build :( :( :(

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [community/community#26087] [Actions]
<p>Trying to post another release, but GitHub Actions has broken again:</p>
<p><a href="https://github.com/ExtendedXmlSerializer/home/runs/625045423?check_suite_focus=true" rel="nofollow noopener">https://github.com/ExtendedXmlSerializer/home/runs/625045423?check_suite_focus=true</a></p>
<p>Why is it so brittle? :(  It worked a month ago.  Is this due to the recent outages?</p>
<p>It is frustrating to put the time into creating these workflows and have them run without issue for months only to have them break with very little explanation or recourse unexpectedly shortly thereafter.</p>
<p>Any assistance here would be greatly appreciated.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/mike-e-wins">@mike-e-wins</a> ,</p>
<p>I also can reproduce the same errors when using your action on my side.</p>
<p>I forked you action repository and tried in the  <strong>Dockerfile</strong> with the following workaround mentioned in some earlier reported issue tickets for the simailar error (<a href="https://github.com/phusion/baseimage-docker/issues/319" rel="nofollow noopener">phusion/baseimage-docker #319</a> and <a href="https://github.com/tianon/docker-brew-ubuntu-core/issues/59" rel="nofollow noopener">docker-brew-ubuntu-core #59</a>),</p>
<pre><code>ARG DEBIAN_FRONTEND=noninteractive
RUN apt-get update &amp;&amp; apt-get install --assume-yes apt-utils
</code></pre>
<p>but it does not work, the errors still exists.</p>
<p>I have reported this ticket to the appropriate engineering team for further investigation and evaluation. If they have any progress, I will notify you in time, and sometimes the appropriate engineers may directly reply you here.</p>

**Comment by Mike-E-angelo** [comment]
<p>Awesome!  Thank you so much for your assistance, <a class="mention" href="/brightran">@brightran</a>!  It is very VERY much appreciated. :+1:</p>

**Comment by miketimofeev** [comment, marked as answer]
<p>Hi <a class="mention" href="/mike-e-wins">@mike-e-wins</a>!</p>
<p>Looks like ubuntu-latest is 20.04, that’s why many of the packages are absent.</p>
<p>Could you please change ubuntu:latest to ubuntu:18.04 in this file?<br>
<a href="https://github.com/DragonSpark/action-changelog-dawg/blob/master/Dockerfile" rel="nofollow noopener">https://github.com/DragonSpark/action-changelog-dawg/blob/master/Dockerfile</a></p>

**Comment by Mike-E-angelo** [comment]
<p>WOOHOO!  It worked!  THANK YOU SO MUCH <a class="mention" href="/miketimofeev">@miketimofeev</a> !  Gratitude not only for fixing my issue but dealing with my lack of patience. :)  My time is limited these days hence my frustration is greater than usual.  I very much appreciate your efforts and time. :+1:</p>

**Comment by Mike-E-angelo** [comment]
<p>Hey all, I was able to push a new build just now and everything worked PERFECTLY!  Thank you all for your diligent and amazing work out there… it is very much appreciated.</p>
<p>Thanks for putting up with my grumpy/snarky remarks in the meantime. :grin::innocent:</p>

**Comment by Mike-E-angelo** [comment]
<p>ALL GREEN THIS IS WHY WE DO THIS :laughing:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/extendedxmlserializer/home/actions/runs/136904189" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2931376/181095160-61f1526f-3858-4a24-bdc8-1c995f17f53b.png" class="thumbnail onebox-avatar" width="323" height="323">

<h3><a href="https://github.com/extendedxmlserializer/home/actions/runs/136904189" target="_blank" rel="nofollow noopener">ExtendedXmlSerializer/home</a></h3>

<p>A highly configurable and eXtensible Xml serializer for .NET. - ExtendedXmlSerializer/home</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Mike-E-angelo** [comment]
<p>It all seems to “just work” now.  Thank you all again for your efforts out there.  They are very much appreciated. :pray:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/ExtendedXmlSerializer/home/actions/runs/213241978" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2931376/181095162-dc48bc24-16c7-400f-8d43-6034b94fcad5.png" class="thumbnail onebox-avatar" width="323" height="323">

<h3><a href="https://github.com/ExtendedXmlSerializer/home/actions/runs/213241978" target="_blank" rel="nofollow noopener">ExtendedXmlSerializer/home</a></h3>

<p>A configurable and eXtensible Xml serializer for .NET. - ExtendedXmlSerializer/home</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
